packages/grafana-ui/src/components/Icon/cached.json

```diff
@@ -160,6 +160,8 @@
   "custom/gf-logs",
   "custom/gf-movepane-left",
   "custom/gf-movepane-right",
+  "custom/gf-prometheus",
```

Comment:
There's no need to add these icons to the js bundle if they will only exist behind a feature flag for now. It'll increase the size of the JS bundle but not be used.

Could you remove them from this list?

Reply:
@academo thank you for your feedback. sure, I can do that.